app/controlplane/pkg/auditor/nats.go

```diff
@@ -27,8 +28,12 @@ import (
 )
 
 const (
-	streamName  = "chainloop-audit"
+	streamName = "chainloop-audit"
+	// subjectName is the base subject for audit logs for the consumer to subscribe to.
```

Comment:
Is `subjectName` used yet?

Reply:
I will check if it's being used by another service. I think I saw something that's why I left it but I'll double check it